Target is not specified - error LNK2019: unresolved external symbol (Windows)

Below issue is caused by the WASM target not being specified. Will raise a PR to fix shortly.

## Steps to repro

1. Clone this repo
1. `cargo build` in server directory

## Result

```shell
C:\dev\repo\temp\Godot-SpacetimeDB-SDK\server (main -> origin)
λ cargo build
   Compiling spacetime-module v0.1.0 (C:\dev\repo\temp\Godot-SpacetimeDB-SDK\server)
warning: unused import: `Rng`
 --> src\lib.rs:9:30
  |
9 |     rand::{seq::SliceRandom, Rng},
  |                              ^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: unused variable: `ctx`
   --> src\lib.rs:164:5
    |
164 |     ctx: &ReducerContext,
    |     ^^^ help: if this is intentional, prefix it with an underscore: `_ctx`
    |
    = note: `#[warn(unused_variables)]` on by default

warning: unused variable: `ctx`
   --> src\lib.rs:219:13
    |
219 | pub fn init(ctx: &ReducerContext) {
    |             ^^^ help: if this is intentional, prefix it with an underscore: `_ctx`

error: linking with `link.exe` failed: exit code: 1120
  |
  = note: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.43.34808\\bin\\HostX64\\x64\\link.exe" "/DEF:C:\\Users\\loki\\AppData\\Local\\Temp\\rustcRtcERg\\lib.def" "/NOLOGO" "C:\\Users\\loki\\AppData\\Local\\Temp\\rustcRtcERg\\symbols.o" "<102 object files omitted>" "C:\\dev\\repo\\temp\\Godot-SpacetimeDB-SDK\\server\\target\\debug\\deps/{libspacetimedb-5aae98a7929c9887.rlib,libspacetimedb_bindings_sys-3a7af4f6680f6fbe.rlib,librand-e96260ad931895d6.rlib,librand_chacha-ac72cc2919ea21e6.rlib,libppv_lite86-ddc08dab81214ed1.rlib,librand_core-edb5d3faa8102c7b.rlib,libgetrandom-60c9ac6d39567f53.rlib,liblog-7f38585503ddb12a.rlib,libspacetimedb_lib-1569b0619e8f5a31.rlib,libblake3-7f31c39f9450d3d0.rlib,libconstant_time_eq-f0ad8e6fb955a741.rlib,libarrayref-ce91e7730d23d527.rlib,libspacetimedb_data_structures-102a2b567bdcd957.rlib,libahash-0b8f4e855e298e1a.rlib,libonce_cell-3f39b4bf050cefb4.rlib,libcfg_if-44ac610e95807e3a.rlib,libzerocopy-0b348eecce155fad.rlib,libhashbrown-be27c063a3d73238.rlib,libequivalent-4ed92b342e2819ee.rlib,libspacetimedb_sats-dc9b1d9499045b73.rlib,libhex-7913ac54bb8295a6.rlib,libarrayvec-8eae33bd231d716f.rlib,libbytes-4303c671e30f4207.rlib,libdecorum-ab4391efcd4ba63d.rlib,libapprox-a96e5ef277ea9219.rlib,libthiserror-d50ad54602970087.rlib,libchrono-46438acf4dff5ea3.rlib,libnum_traits-068299fbe3249c24.rlib,libsha3-6f473bd6c5efe0e5.rlib,libkeccak-c89c2f2b7ee3d1a5.rlib,libdigest-558a1be628a6799c.rlib,libblock_buffer-faa17de81cd2b9c0.rlib,libcrypto_common-3e6576fa86132820.rlib,libgeneric_array-fcf09f8633aa2945.rlib,libtypenum-17c2d090c3bb66ad.rlib,libspacetimedb_primitives-2c76659350072032.rlib,libnohash_hasher-d28059e493038133.rlib,libbitflags-2274350df5803888.rlib,libitertools-6a667d900204accd.rlib,libeither-94232329d545e34c.rlib,libsmallvec-c992ea4c5760f9cc.rlib,libbytemuck-3b4a6c86eacb614d.rlib,libethnum-014a7eb311f8e3d5.rlib,libserde-c49825f9db3e5454.rlib,libsecond_stack-44ee6b0ae0059ccb.rlib,libanyhow-06198aaeaa040d6e.rlib}" "C:\\Users\\loki\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib/{libstd-99a5467416e27682.rlib,libpanic_unwind-b2376f55ca9ba5db.rlib,libwindows_targets-e982c2634e026463.rlib,librustc_demangle-622607d70f9500c2.rlib,libstd_detect-402c0b1e8e67cb33.rlib,libhashbrown-d36035ae19bb0c8b.rlib,librustc_std_workspace_alloc-cb12c14614b55d43.rlib,libunwind-8e666946f8f5db2b.rlib,libcfg_if-ac9947000bdc169e.rlib,liballoc-4f54ad4ac4b0f4c5.rlib,librustc_std_workspace_core-ff8db640d177ed2a.rlib,libcore-745ff350a54e4299.rlib,libcompiler_builtins-b2f39c5f2779068f.rlib}" "bcrypt.lib" "advapi32.lib" "kernel32.lib" "kernel32.lib" "advapi32.lib" "ntdll.lib" "userenv.lib" "ws2_32.lib" "dbghelp.lib" "/defaultlib:msvcrt" "/NXCOMPAT" "/LIBPATH:C:\\dev\\repo\\temp\\Godot-SpacetimeDB-SDK\\server\\target\\debug\\build\\blake3-d099f01c63a2e9dc\\out" "/LIBPATH:C:\\dev\\repo\\temp\\Godot-SpacetimeDB-SDK\\server\\target\\debug\\build\\blake3-d099f01c63a2e9dc\\out" "/OUT:C:\\dev\\repo\\temp\\Godot-SpacetimeDB-SDK\\server\\target\\debug\\deps\\spacetime_module.dll" "/OPT:REF,NOICF" "/DLL" "/IMPLIB:C:\\dev\\repo\\temp\\Godot-SpacetimeDB-SDK\\server\\target\\debug\\deps\\spacetime_module.dll.lib" "/DEBUG" "/PDBALTPATH:%_PDB%" "/NATVIS:C:\\Users\\loki\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\intrinsic.natvis" "/NATVIS:C:\\Users\\loki\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\liballoc.natvis" "/NATVIS:C:\\Users\\loki\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libcore.natvis" "/NATVIS:C:\\Users\\loki\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libstd.natvis"
  = note: some arguments are omitted. use `--verbose` to show all linker arguments
  = note:    Creating library C:\dev\repo\temp\Godot-SpacetimeDB-SDK\server\target\debug\deps\spacetime_module.dll.lib and object C:\dev\repo\temp\Godot-SpacetimeDB-SDK\server\target\debug\deps\spacetime_module.dll.exp␍
          spacetime_module.3o622st1ki11zoxuojn8n3vbp.rcgu.o : error LNK2019: unresolved external symbol index_id_from_name referenced in function _ZN24spacetimedb_bindings_sys18index_id_from_name28_$u7b$$u7b$closure$u7d$$u7d$17hb8110a597ec87c73E␍
          spacetime_module.78sx9tszq8qw3arzvx8papgmd.rcgu.o : error LNK2019: unresolved external symbol datastore_insert_bsatn referenced in function _ZN24spacetimedb_bindings_sys22datastore_insert_bsatn17h360318a8bd759860E␍
          spacetime_module.78sx9tszq8qw3arzvx8papgmd.rcgu.o : error LNK2019: unresolved external symbol datastore_update_bsatn referenced in function _ZN24spacetimedb_bindings_sys22datastore_update_bsatn17ha1e5b30c454445b4E␍
          libspacetimedb-5aae98a7929c9887.rlib(spacetimedb-5aae98a7929c9887.spacetimedb.13d1f014490d29c4-cgu.3.rcgu.o) : error LNK2019: unresolved external symbol console_timer_start referenced in function _ZN11spacetimedb13log_stopwatch12LogStopwatch3new17h320c93b9c0d55ea0E␍
          libspacetimedb-5aae98a7929c9887.rlib(spacetimedb-5aae98a7929c9887.spacetimedb.13d1f014490d29c4-cgu.3.rcgu.o) : error LNK2019: unresolved external symbol console_timer_end referenced in function _ZN82_$LT$spacetimedb..log_stopwatch..LogStopwatch$u20$as$u20$core..ops..drop..Drop$GT$4drop17hf81fdf543dd8d9f3E␍
          libspacetimedb-5aae98a7929c9887.rlib(spacetimedb-5aae98a7929c9887.spacetimedb.13d1f014490d29c4-cgu.3.rcgu.o) : error LNK2019: unresolved external symbol bytes_source_read referenced in function _ZN11spacetimedb2rt22read_bytes_source_into17h39d40aab51ec3e82E␍
          libspacetimedb-5aae98a7929c9887.rlib(spacetimedb-5aae98a7929c9887.spacetimedb.13d1f014490d29c4-cgu.3.rcgu.o) : error LNK2019: unresolved external symbol bytes_sink_write referenced in function _ZN11spacetimedb2rt13write_to_sink17h549472dbb3988407E␍
          libspacetimedb-5aae98a7929c9887.rlib(spacetimedb-5aae98a7929c9887.spacetimedb.13d1f014490d29c4-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol console_log referenced in function _ZN24spacetimedb_bindings_sys11console_log17h40accd3bd93121d4E␍
          libspacetimedb-5aae98a7929c9887.rlib(spacetimedb-5aae98a7929c9887.spacetimedb.13d1f014490d29c4-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol table_id_from_name referenced in function _ZN24spacetimedb_bindings_sys18table_id_from_name28_$u7b$$u7b$closure$u7d$$u7d$17h6caf14a480432927E␍
          libspacetimedb-5aae98a7929c9887.rlib(spacetimedb-5aae98a7929c9887.spacetimedb.13d1f014490d29c4-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol identity referenced in function _ZN24spacetimedb_bindings_sys8identity17h155a5e862d0f6caeE␍
          libspacetimedb_bindings_sys-3a7af4f6680f6fbe.rlib(spacetimedb_bindings_sys-3a7af4f6680f6fbe.spacetimedb_bindings_sys.65fcc8568a1e022-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol datastore_table_scan_bsatn referenced in function _ZN24spacetimedb_bindings_sys26datastore_table_scan_bsatn28_$u7b$$u7b$closure$u7d$$u7d$17h2d92642d8d2507c3E␍
          libspacetimedb_bindings_sys-3a7af4f6680f6fbe.rlib(spacetimedb_bindings_sys-3a7af4f6680f6fbe.spacetimedb_bindings_sys.65fcc8568a1e022-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol datastore_index_scan_range_bsatn referenced in function _ZN24spacetimedb_bindings_sys32datastore_index_scan_range_bsatn28_$u7b$$u7b$closure$u7d$$u7d$17h7e02034652ae97f5E␍
          libspacetimedb_bindings_sys-3a7af4f6680f6fbe.rlib(spacetimedb_bindings_sys-3a7af4f6680f6fbe.spacetimedb_bindings_sys.65fcc8568a1e022-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol datastore_delete_by_index_scan_range_bsatn referenced in function _ZN24spacetimedb_bindings_sys42datastore_delete_by_index_scan_range_bsatn28_$u7b$$u7b$closure$u7d$$u7d$17h9e44ce338b394b7bE␍
          libspacetimedb_bindings_sys-3a7af4f6680f6fbe.rlib(spacetimedb_bindings_sys-3a7af4f6680f6fbe.spacetimedb_bindings_sys.65fcc8568a1e022-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol row_iter_bsatn_advance referenced in function _ZN24spacetimedb_bindings_sys7RowIter4read17h349c3963c02c50b5E␍
          libspacetimedb_bindings_sys-3a7af4f6680f6fbe.rlib(spacetimedb_bindings_sys-3a7af4f6680f6fbe.spacetimedb_bindings_sys.65fcc8568a1e022-cgu.0.rcgu.o) : error LNK2019: unresolved external symbol row_iter_bsatn_close referenced in function _ZN75_$LT$spacetimedb_bindings_sys..RowIter$u20$as$u20$core..ops..drop..Drop$GT$4drop17hde2deb975a029908E␍
          C:\dev\repo\temp\Godot-SpacetimeDB-SDK\server\target\debug\deps\spacetime_module.dll : fatal error LNK1120: 15 unresolved externals␍


warning: `spacetime-module` (lib) generated 3 warnings
error: could not compile `spacetime-module` (lib) due to 1 previous error; 3 warnings emitted
```

--- a/server/.cargo/config.toml
+++ b/server/.cargo/config.toml
@@ -1,0 +1,2 @@
+[build]
+target = "wasm32-unknown-unknown"

